BlueprintUI/Sources/Layout/Stack.swift

```diff
@@ -146,6 +146,15 @@ extension StackLayout {
 
         /// Additional space will be distributed uniformly between children.
         case growUniformly
+
+        /// Additional space will appear after all children.
+        case justifyStart
+
+        /// Additional space will be distributed on either side of all children.
+        case justifyMiddle
+
+        /// Additional space will be appear before all children.
+        case justifyEnd
```

Comment:
My concern with using the word "align" is that that word is already used for the cross-axis. I agree about center and using "to". I chose "justify" because that's the standard terminology used by Flexbox, React, and ComponentKit, but if there's another word we can think of I'm def open to it!

Reply:
Hm, well I guess I disagree with that terminology choice (I think it's fine if both axes have alignment), but certainly not strongly enough to block this.

Reply:
Can we just drop the prefix? `.start` `.center` `.end`?

Reply:
I was chatting with @bencochran last night and and we came up with `.trailingSpace`, `.center`, and `.leadingSpace`. What are your guys' thoughts?

Reply:
I don't like the inconsistent grammar, personally. `spaceEvenly`, `growUniformly` etc are imperative verb phrases that say what to do with the child elements. They fit in the sentence "the elements will ____". I'd choose the `justifyToX` variants over these.

Reply:
Yeah I think I agree. It's weird to focus on the space instead of the content since the content is what we care about